fuse/plugins/micro_physics/lineage_cluster.py

```diff
@@ -393,14 +467,13 @@ def is_lineage_broken(
 
         particle_position = np.array([particle["x"], particle["y"], particle["z"]])
         parent_position = np.array([parent["x"], parent["y"], parent["z"]])
-
         distance = np.sqrt(np.sum((parent_position - particle_position) ** 2, axis=0))
 
-        if (particle["creaproc"] == "phot") and (particle["edproc"] == "phot"):
-            # we do not want to split a photo absorption into two clusters
-            # the second photo absorbtion (that we see) could be x rays
-            # so, do not split if the distance is small
-            return False
+        if particle["creaproc"] == "eBrem":
+            # we do not want to split a bremsstrahlung into two clusters
+            # if the distance is really small, it is most likely the same interaction
+            if distance < 0.1:  # cm
+                return False
```

Comment:
I think we should put this into a URLConfig to keep track of it.

Reply:
Added :)